src/scss/modules/list-pages.scss

```diff
@@ -15,18 +15,24 @@
 }
 
 .list-page__title {
+  align-items: center;
   border-bottom: 1px solid $black-90;
   font-size: 24px;
   margin-bottom: 16px;
   padding-bottom: 16px;
 }
 
+.list-page__owner {
```

Comment:
this is another style in a module.. I think "owner" is too domain for this.  needs to be made more generic

Reply:
maybe `list-page__title-filter`?  I know it's not always a filter though..

Reply:
Works for me 😄 

Updated that in what I just pushed.